torch/_inductor/cpp_builder.py

```diff
@@ -196,6 +196,33 @@ def _is_msvc_cl(cpp_compiler: str) -> bool:
     return False
 
 
+@functools.lru_cache(None)
+def _is_intel_compiler(cpp_compiler: str) -> bool:
```

Comment:
Do we expect it works on Linux too or only on Windows?

Reply:
@jgong5 I think it also working on Linux, or just a small gap to fill.